rules/S3518/metadata.json

```
@@ -3,7 +3,8 @@
   "type": "BUG",
   "code": {
     "impacts": {
-      "RELIABILITY": "HIGH"
+      "RELIABILITY": "HIGH",
+      "SECURITY": "HIGH"
```

> Zero should not be a possible denominator

```suggestion
      "RELIABILITY": "HIGH",
      "SECURITY": "MEDIUM"
```

Following CWE assessment.